### DISCUSSION TITLE
Issue with i18n.date and i18n.number in Lingui v3.x

### DISCUSSION THREAD

**Discussion by olajide1234** [lingui/js-lingui#1086] [Q&A]
Hi. I'm working on migrating our project to Lingui v3.x, but I have an issue with i18n.date and i18n.number.  i18n.date basically does not format the date but returns something like `value,date,date0`. 

It's probably an issue with our project, because when I create a minimal code sandbox, i18n works fine. I think @semoal has reported something similar with a server-side rendered app, but did not disclose the cause. Our app is not server-side rendered.

Here is an example use case for us:

```
import React from 'react';
import { i18n } from '@lingui/core';

export default function SampleDateFunction({ value, format, ...rest }) {

  return (
    <React.Fragment>
      {i18n.date(value, { ...format })}
    </React.Fragment>
  );
}
```

By the way, if I create a reference to the date function e.g. i18n.MyOwnDateRef = i18n.date; and then use ` i18n.MyOwnDateRef` in our app; that works fine.

I'ld be glad to get any suggestions.

**Comment by semoal** [comment]
Hello @olajide1234 nice to see that you're trying to migrate to v3 branch, and I'm happy to help.

Could you create a repo with next-js, and something similar to your setup where i can reproduce the issue? Not too much free time with Rematch, writing a book, maintaining Lingui, my daily work, etc... hehe
